File: fizzbuzz.py
# FizzBuzz関数
def fizzbuzz(n):
    if n % 15 == 0:
        return "FizzBuzz"
    if n % 3 == 0:
        return "Fizz"
    if n % 5 == 0:
        return "Buzz"
    return n

# 1〜20を出力
if __name__ == "__main__":
    for i in range(1, 21):
        print(fizzbuzz(i))

File: fizzbuzz.js
// FizzBuzz関数
function fizzbuzz(n) {
  const divisibleBy3 = n % 3 === 0;
  const divisibleBy5 = n % 5 === 0;
  
  if (divisibleBy3 && divisibleBy5) return "FizzBuzz";
  if (divisibleBy3) return "Fizz";
  if (divisibleBy5) return "Buzz";
  return n;
}

// 1〜20を出力する関数
function printFizzBuzz(max = 20) {
  for (let i = 1; i <= max; i++) {
    console.log(fizzbuzz(i));
  }
}

// スクリプトとして直接実行された場合のみ出力
if (require.main === module) {
  printFizzBuzz();
}

module.exports = { fizzbuzz, printFizzBuzz };
